osiguranje na klik/main.css

```
@@ -664,6 +673,247 @@ img {
     overflow: hidden;
 }
 
+.blog-hero-bg {
+    background-color: #F9F5F2;    
+    background-image: url(img/pattern-green.png);
+}
+
+.blog-hero {
+    text-align: center;    
+    padding: 80px 24px; 
+    position: relative;   
+}
+
+.blog-hero::after {
+    position: absolute;
+    display: block;    
+    content: url(img/blog-hero-dots-pink.svg);
+    bottom: 20px;    
+}
+
+.blog-navigation {
+    padding: 10px 24px; 
+    border-bottom: 1px solid #E3E6E5;  
+    position: relative; 
+}
+
+.blog-navigation-large {
+    padding: 30px 40px 0 40px;
+    border-bottom: 1px solid #E3E6E5;
+    display: none;
+    justify-content: space-between;
+    font-size: 13px;
+    color:  #9F9FA8;    
+}
+
+.blog-navigation-large a {
+    padding-bottom: 30px;     
+    text-transform: uppercase;   
+}
+
+.blog-navigation-large a:hover {
+    border-bottom: 2px solid #088E8F;
+    color: #088E8F;
+    padding-bottom: 28px;
+}
```

Nema potrebe da menjas `padding`. To radis zato sto imas gore `border-bottom` ali to moze bolje da se resi. Tako sto stavis na link da uvek postoji `border-bottom: 2px solid transparent;`, znaci uvek postoji ivica od `2px` samo sto je provedna. Kad se uradi hover promeni se boja `border-bottom-color` ili `border-color`

```suggestion
.blog-navigation-large a {
    padding-bottom: 30px;
    border-bottom: 2px solid #088E8F;     
    text-transform: uppercase;   
}

.blog-navigation-large a:hover {
    border-bottom-color: #088E8F;
    color: #088E8F;
}
```